### DISCUSSION TITLE
save model

### DISCUSSION THREAD

**Discussion by bondio77** [asteroid-team/asteroid#570] [Q&A] [answered]
hi, i want to save the model which finish the training but i can't find the save function.
 how could i save the model?
this is second time to training, the first one is basic cnn with mnist dataset
Thank you

**Comment by mpariente** [comment, marked as answer]
It's called `serialize`, do you find it?

**Reply by bondio77** [reply]
i find it, thank you!
but i already use 'torch.save(model)' and now still training. i will use serialize function at next training.
and this is another question, is there no problem about save the model using torch? not serialize? (in this asteroid)
thank you so much.

**Reply by bondio77** [reply]
sorry for my weird question i just got to know how to save model .
Thanks a lot!
